app/views/hotels/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1> <%= @hotel.name %></h1>
+<div id="hotel">
```

Comment:
Cool to use a div here; you can also assign the `<ul>` element an ID, to save some typing.